src/main/java/org/folio/marc4ld/service/marc2ld/field/property/builder/MappingPropertyBuilder.java

```diff
@@ -1,31 +0,0 @@
-package org.folio.marc4ld.service.marc2ld.field.property.builder;
-
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Optional;
-import lombok.RequiredArgsConstructor;
-import org.apache.commons.lang3.StringUtils;
-import org.folio.marc4ld.service.dictionary.DictionaryProcessor;
-import org.folio.marc4ld.service.marc2ld.field.property.Property;
-import org.marc4j.marc.DataField;
-import org.marc4j.marc.Subfield;
-
-@RequiredArgsConstructor
-public class MappingPropertyBuilder implements PropertyBuilder<DataField> {
```

Comment:
As I understand, we removed it because it's used only in 1 place?

Reply:
Yes. It is used only in one place